lightkurve/interact.py

```diff
@@ -263,8 +264,18 @@ def stretch_change_callback(attr, old, new):
 
     return fig, stretch_slider
 
+def make_default_export_name(tpf, suffix='custom-aperture-mask'):
```

Comment:
Should this be `suffix='custom-lc'`, given that the button is creating a light curve file with aperture mask extension?

Reply:
see comment above.